project/build.properties

```diff
@@ -1 +1 @@
-sbt.version = 1.5.7
+sbt.version = 1.8.2
```

Comment:
Can we have a newline at the end for consistency?

Reply:
not sure what you mean